src/app/ASKS/ask/index.js

```diff
@@ -731,6 +732,28 @@ const Ask = ({
           clientId={clientId}
         />
       )}
+      {component === 'repeatable_dropdown' && (
```

Comment:
Like the checkboxes, the **'repeatable_dropdown'** can be a type whereas it needs to be handled inside of dropdown component